pkg/operator/ceph/pool/controller.go

```diff
@@ -39,6 +39,7 @@ const (
 	customResourceNamePlural = "pools"
 	replicatedType           = "replicated"
 	erasureCodeType          = "erasure-coded"
+	appName                  = "rbd"
```

Comment:
Could you rename that to be more "fitting" even if used only here, e.g. `poolApplicationNameRBD`.

Reply:
Renamed.